### ISSUE TITLE
vmdkops_admin output is not consistent when more than 2 ESX are present and accessing shared datastore (attached-to-vm field is inconsistent)

### ISSUE THREAD

**Issue by shuklanirdesh82** [vmware/docker-volume-vsphere#934]
vmdkops_admin output is not consistent when more than 2 ESX are present and accessing shared datastore e.g. vsan, shared vmfs etc (attached-to-vm field is inconsistent)

Test bed scenario: 2 ESX having shared datastore (vsan/shared vmfs). One docker host per ESX created on the the shared datastore (let's say vm1 & vm2)

Note: both VMs are part of _DEFAULT tenant

Steps to reproduce
1. create volume from vm1 & vm2 on shared datastore
```
VM1: docker volume create --driver=vmdk --name=vol_231_vsan
VM2: docker volume create --driver=vmdk --name=vol_99_vsan
```
2. make sure both volumes are accessible from either VMs (docker volume ls from both VMs)
3. attach container from one of the VM 
```
VM1: docker run --rm -it -v vol_231_vsan@vsanDatastore:/mnt/myvol2 --name cnt_2311 busybox
```
4. invoke docker inspect from another VM and make sure it says volume is ```attached```
```
root@photon-TX4FfW2ET [ ~ ]# docker volume inspect vol_231_vsan@vsanDatastore
[
    {
        "Name": "vol_231_vsan@vsanDatastore",
        "Driver": "vmdk",
        "Mountpoint": "/mnt/vmdk/vol_231_vsan@vsanDatastore",
        "Status": {
            "access": "read-write",
            "attach-as": "independent_persistent",
            "capacity": {
                "allocated": "80MB",
                "size": "100MB"
            },
            "clone-from": "None",
            "created": "Sat Feb 18 06:16:43 2017",
            "created by VM": "VM84_vsan",
            "datastore": "vsanDatastore",
            "diskformat": "thin",
            "fstype": "ext4",
            "status": "attached"
        },
        "Labels": {},
        "Scope": "global"
    }
]
```

5. On ESX side invoke vmdkops_admin

```
ESX1:

[root@sc-rdops-vm17-dhcp-0-84:/vmfs/volumes] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py ls
Volume          Datastore      Created By VM  Created                   Attached To VM  Policy          Capacity  Used     Disk Format  Filesystem Type  Access      Attach As               
--------------  -------------  -------------  ------------------------  --------------  --------------  --------  -------  -----------  ---------------  ----------  ----------------------   
vol_231_vsan    vsanDatastore  VM84_vsan      Sat Feb 18 06:16:43 2017  VM84_vsan       [VSAN default]  100.00MB  80.00MB  thin         ext4             read-write  independent_persistent  
vol_99_vsan     vsanDatastore  VM100_vsan     Sat Feb 18 06:20:54 2017  detached        [VSAN default]  100.00MB  80.00MB  thin         ext4             read-write  independent_persistent 

ESX2:

[root@sc-rdops-vm17-dhcp-15-100:/vmfs/volumes] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py ls
Volume          Datastore      Created By VM  Created                   Attached To VM  Policy          Capacity  Used     Disk Format  Filesystem Type  Access      Attach As               
--------------  -------------  -------------  ------------------------  --------------  --------------  --------  -------  -----------  ---------------  ----------  ----------------------  
vol_231_vsan    vsanDatastore  VM84_vsan      Sat Feb 18 06:16:43 2017  detached        [VSAN default]  100.00MB  80.00MB  thin         ext4             read-write  independent_persistent  
vol_99_vsan     vsanDatastore  VM100_vsan     Sat Feb 18 06:20:54 2017  detached        [VSAN default]  100.00MB  80.00MB  thin         ext4             read-write  independent_persistent  
```

**Expected outcome**: admin ls outcome should be consistent from both ESX hosts.

**Comment by govint** [comment]
Issue is on the second ESX host, the VM UUID isn't known and hence printing as detached. Instead if the VM UUID isn't known then should instead pring the UUID itself. At least that way user has a way to know the volume is in use. Plus we should print the VM as <vm name/uuid>@<local/hostname>.

**Comment by govint** [comment]
Fixed via #939
